exercises/pig-latin/description.md

```diff
@@ -5,10 +5,12 @@ confusing. It obeys a few simple rules (below), but when it's spoken
 quickly it's really difficult for non-children (and non-native speakers)
 to understand.
 
-- **Rule 1**: If a word begins with a vowel sound, add an "ay" sound to
-  the end of the word.
-- **Rule 2**: If a word begins with a consonant sound, move it to the
+* **Rule 1**: If a word begins with a vowel sound, add an "ay" sound to
```

Comment:
Please retain `-` bullets throughout instead of switching to `*` bullets. Marked here only, but applies to all four rules.

Reply:
No problem. I can fix this, overlooked the markdown as the output looked the same.